docs/user/three_dof_simulation.rst

```diff
@@ -340,6 +340,316 @@ Here's a complete 3-DOF simulation from start to finish:
 
     flight.plots.trajectory_3d()
 
+Weathercocking Model
+--------------------
+
+RocketPy's 3-DOF simulation mode includes a weathercocking model that allows
+the rocket's attitude to evolve during flight. This feature simulates how a
+statically stable rocket naturally aligns with the relative wind direction.
+
+Understanding Weathercocking
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Weathercocking is the tendency of a rocket to align its body axis with the
+direction of the relative wind. In reality, this occurs due to aerodynamic
+restoring moments from fins and other stabilizing surfaces. The 3-DOF
+weathercocking model provides a simplified representation of this behavior
+without requiring full 6-DOF rotational dynamics.
+
+The ``weathercock_coeff`` Parameter
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The weathercocking behavior is controlled by the ``weathercock_coeff`` parameter
+in the :class:`rocketpy.Flight` class:
+
+.. jupyter-execute::
+
+    from rocketpy import Environment, PointMassMotor, PointMassRocket, Flight
+
+    env = Environment(
+        latitude=32.990254,
+        longitude=-106.974998,
+        elevation=1400
+    )
+    env.set_atmospheric_model(type="StandardAtmosphere")
+
+    motor = PointMassMotor(
+        thrust_source=1500,
+        dry_mass=1.5,
+        propellant_initial_mass=2.5,
+        burn_time=3.5,
+    )
+
+    rocket = PointMassRocket(
+        radius=0.078,
+        mass=15.0,
+        center_of_mass_without_motor=0.0,
+        power_off_drag=0.43,
+        power_on_drag=0.43,
+    )
+    rocket.add_motor(motor, position=0)
+
+    # Flight with weathercocking enabled
+    flight = Flight(
+        rocket=rocket,
+        environment=env,
+        rail_length=4.2,
+        inclination=85,
+        heading=45,
+        simulation_mode="3 DOF",
+        weathercock_coeff=1.0,  # Default value
+    )
+
+    print(f"Apogee: {flight.apogee - env.elevation:.2f} m")
+
+The ``weathercock_coeff`` parameter controls the rate at which the rocket
+aligns with the relative wind:
+
+- ``weathercock_coeff=0``: No weathercocking (original fixed-attitude behavior)
+- ``weathercock_coeff=1.0``: Default moderate alignment rate
+- ``weathercock_coeff>1.0``: Faster alignment (more stable rocket)
+
+Effect of Weathercocking Coefficient
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Higher values of ``weathercock_coeff`` result in faster alignment with the
+relative wind. This affects the lateral motion and impact point:
+
+.. list-table:: Weathercocking Coefficient Effects
```

Comment:
Out of curiosity, what is the source of these values?

```rst
.. list-table:: Weathercocking Coefficient Effects
```

Reply:
That's alright, I understand it.

But I mean, where did you learn this from? Can you site your sources?

This is important (not mandatory) to users that want to use this new kind of simulation